README.md

```diff
@@ -233,6 +276,9 @@ arbitrary settings, which means the following four characters:
 - `,` is `%2C`
 - `}` is `%7D`
 - ` ` is `%20`
+- `[` is `%5B`
+- `]` is `%5D`
+- `;` is `%3B`
```

Comment:
We should delete the word "four" from line 273 so we don't have to update the count every time we update this list (and the count doesn't help anything anyway).